src/JSInterop/Microsoft.JSInterop/src/JSRuntime.cs

```diff
@@ -191,5 +195,29 @@ internal void EndInvokeJS(long taskId, bool succeeded, ref Utf8JsonReader jsonRe
                 TaskGenericsUtil.SetTaskCompletionSourceException(tcs, new JSException(message, exception));
             }
         }
+
+        #region DotNetObjectReference
```

Comment:
meh. We don't typically use regions.

Reply:
After getting to the tests, I see why you put this in a region. I left a suggestion elsewhere about this.